sabnzbd/api.py

```diff
@@ -1798,9 +1798,9 @@ def build_header(webdir: str = "", for_template: bool = True, trans_functions: b
     header["have_warnings"] = str(sabnzbd.GUIHANDLER.count())
     header["finishaction"] = sabnzbd.QUEUECOMPLETE
 
-    header["quota"] = to_units(sabnzbd.BPSMeter.quota)
+    header["quota"] = to_units(sabnzbd.BPSMeter.quota, "B")
     header["have_quota"] = bool(sabnzbd.BPSMeter.quota > 0.0)
-    header["left_quota"] = to_units(sabnzbd.BPSMeter.left)
```

Comment:
It isn't that simple, otherwise I would have fixed it long ago.

You also need to change it in the front end where we use this value to give a visual indicator if the quote was exceeded.

Reply:
> You also need to change it in the front end

Apologies. I tried to follow the discussion on the issue because it did indeed seem too simple.  I apparently did not follow it as well as I thought.  I took away the fact that it produces strings instead of numbers was a hassle, but meant that its output would be usable.

I will look into this modulo the other comments which have come in.

Reply:
@Safihre This commit now actually (and finally) includes the UI changes.  The tests seem to be passing.

I searched my best for places where the values are used and updated them as necessary. I've looked in the UI and not noticed any anomalies. I do not, however, doubt my human ability to have missed something.

Reply:
Nice!